File: 1_forloop_add_test/forloop_add_test.m
#!/usr/bin/env -S octave -q

felesleges_de_kell_ide_egy_ertekadas = 0;

function main(toval)
    sumval = 0;
    for i = 0 : toval-1
        for j = 0 : toval-1
            sumval = sumval + i+(j^1);
        endfor
    endfor
    printf("Octave: %d\n", sumval);
end

main( str2num(argv(){1}) );

File: 1_forloop_add_test/src/forloop_add_test.nim
#!/usr/bin/env -S nim c -r

import os
import strutils

proc main(toval: int) =
    var sumval = 0;
    for i in 0..toval-1:
        for j in 0..toval-1:
            sumval += i+(j xor 1)

    echo "Nim:", sumval

main( parseInt( paramStr(1) ) )
